source/documentation/internal/guest-agent/architecture.html.md

```diff
@@ -14,16 +14,17 @@ wiki_last_updated: 2013-05-20
 
 ## The communication between oVirt guest agent and the oVirt engine
 
-![](Guest_Agent_Achitecture.png "Guest_Agent_Achitecture.png")
+<!-- TODO: image too large, needs fixing
```

Comment:
No I can do that. It's just that I'm hessitant whether to keep the image at all. But I will shrink it and we can remove it later if we want.